<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_User</class>
 <widget class="QWidget" name="Tab_User">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableView" name="table_user">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetButtom" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="15,2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_delete">
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_add">
        <property name="text">
         <string>增加</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_change">
        <property name="text">
         <string>修改</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="QWidget" name="widgetFilter_2" native="true">
        <layout class="QFormLayout" name="widgetFilter">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit_user_id"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_user_name">
           <property name="text">
            <string>用户姓名查询：</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit_user_name"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_user_authority">
           <property name="text">
            <string>用户权限查询：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBox_user_authority">
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Level_1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Level_2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Level_3</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_user_register_time">
           <property name="text">
            <string>用户注册时间查询：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_user_id">
           <property name="text">
            <string>用户id查询：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit_user_register_time">
           <property name="placeholderText">
            <string>yyyy/M/d hh:mm</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
